pkg/controller/clusterinstallation/mattermost.go

```diff
@@ -192,24 +192,20 @@ func (r *ReconcileClusterInstallation) checkMattermostDeployment(mattermost *mat
 // If an update is required then the deployment spec is set to:
 // - roll forward version
 // - keep active MattermostInstallation available by setting maxUnavailable=N-1
-func (r *ReconcileClusterInstallation) updateMattermostDeployment(mi *mattermostv1alpha1.ClusterInstallation, original, d *appsv1.Deployment, reqLogger logr.Logger) error {
+func (r *ReconcileClusterInstallation) updateMattermostDeployment(mi *mattermostv1alpha1.ClusterInstallation, modified, original *appsv1.Deployment, reqLogger logr.Logger) error {
```

Comment:
nit: maybe the name `new` instead of `modified` makes sense here?

Reply:
done